<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ng_pluginDockWidgetBase</class>
 <widget class="QDockWidget" name="ng_pluginDockWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>265</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Locus QGIS - Search</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="searchWidget" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="settingsButton">
           <property name="text">
            <string>Settings</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QLabel" name="Mode">
           <property name="text">
            <string>Mode</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="modeCombo">
           <item>
            <property name="text">
             <string comment="Catergory Search" extracomment="Catergory Search">category_search</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string comment="Bounding Box" extracomment="Bounding Box">bounding_box</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string comment="Reference Search" extracomment="Reference Search">reference_search</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string comment="Point Search" extracomment="Point Search">point_search</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="categoryLabel">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>10</height>
            </size>
           </property>
           <property name="text">
            <string>Category</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="categoryCombo"/>
         </item>
         <item>
          <widget class="QLabel" name="searchLabel">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>10</height>
            </size>
           </property>
           <property name="text">
            <string>Search Text</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="searchField">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100000</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Normal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="horizontalWidget" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <property name="topMargin">
             <number>1</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="runButton">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>172417</height>
               </size>
              </property>
              <property name="text">
               <string>Run</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
